api/src/models/team.model.js

```diff
@@ -22,14 +22,6 @@ const teamSchema = new mongoose.Schema(
       type: String,
       required: true,
     },
-    areaOfInterest: {
```

Comment:
I might have missed something... why do we want to remove this?

Reply:
I think I also missed the discussion. I just refer to the model in this link: https://www.notion.so/Researchify-DB-Schema-78d164ce0c6147f9831f79cd9b89c519